dolphinscheduler-task-plugin/dolphinscheduler-task-java/src/main/java/org/apache/dolphinscheduler/plugin/task/java/JavaParameters.java

```diff
@@ -68,7 +69,7 @@ public class JavaParameters extends AbstractParameters {
      */
     @Override
     public boolean checkParameters() {
-        return runType != null && (rawScript != null && !rawScript.isEmpty()) || mainJar != null;
+        return runType != null && StringUtils.isNotBlank(rawScript) || mainJar != null;
```

Comment:
```suggestion
        return runType != null && (StringUtils.isNotBlank(rawScript) || mainJar != null);
```

When the runType is jar you need to check if the `mainJar != null`, when the runType is script you need to check if the `rawScript != null`